packages/read-and-create-calendar-events/client/react/src/App.js

```diff
@@ -140,33 +148,45 @@ function Agenda({ serverBaseUrl, userId, calendarId }) {
 
   useEffect(() => {
     const getCalendarEvents = async () => {
-      try {
-        const url = `${serverBaseUrl}/nylas/read-events?${
-          calendarId ? 'calendarId=' + calendarId : ''
-        }`;
-
-        const res = await fetch(url, {
-          method: 'GET',
-          headers: {
-            Authorization: userId,
-            'Content-Type': 'application/json',
-          },
-          params: {
-            calendarId,
-          },
-        });
-
-        if (!res.ok) {
-          throw new Error(res.statusText);
+      if (calendarId) {
+        try {
+          const startsAfter = null; // applyTimezone(getLocalDateString(new Date(new Date().setDate(new Date().getDate() - 7))));
+          const endsBefore = applyTimezone(
+            getLocalDateString(
+              new Date(new Date().setDate(new Date().getDate() + 7))
+            )
+          );
+          const limit = 5;
+
+          const url = `${serverBaseUrl}/nylas/read-events?${
+            calendarId ? 'calendarId=' + calendarId : ''
+          }${startsAfter ? '&startsAfter=' + startsAfter : ''}${
+            endsBefore ? '&endsBefore=' + endsBefore : ''
+          }${limit ? '&limit=' + limit : ''}`;
```

Comment:
Can we make this more readable? The url can be static, we don't need it don't need it to be fancy. Eg. `startsAfter` can resolve from a util function like `getTodaysDate` and `endsBefore` can be `getSevenDaysLater`. Hard code `limit` as well.